src/esp/gfx/RenderCamera.h

```diff
@@ -104,14 +104,30 @@ class RenderCamera : public MagnumCamera {
    */
   ~RenderCamera() override = default;
 
-  // Get the scene node being attached to.
+  /**
+   * @brief Get the scene node being attached to.
+   * @return the current scene node
+   */
   scene::SceneNode& node() { return object(); }
+
+  /**
+   * @brief Get a const ref to the scene node being attached to.
+   * @return the current scene node
+   */
   const scene::SceneNode& node() const { return object(); }
 
-  // Overloads to avoid confusion
+  /**
+   * @brief Overloads to avoid confusion
+   * @return ref to current scene node
+   */
   scene::SceneNode& object() {
     return static_cast<scene::SceneNode&>(MagnumCamera::object());
   }
+
+  /**
+   * @brief Overloads to avoid confusion
+   * @return ref to current scene node
+   */
```

Comment:
I'd just do `/** @overload */` for the two here. Less typing and Doxygen puts some prefabricated docs to these.